pkg/kubectl/delete.go

```diff
@@ -404,6 +404,10 @@ func (reaper *DeploymentReaper) Stop(namespace, name string, timeout time.Durati
 	if err != nil {
 		return err
 	}
+	if deployment.Initializers != nil {
+		deleteOption := metav1.DeleteOptions{}
```

Comment:
Please use the nonOprhan option like the other place in this fucntion
 ```
nonOrphanOption := metav1.DeleteOptions{OrphanDependents: &falseVar}
```
Otherwise apiserver will add a finalizer on the deployment and garbage collector won't see it, because the deployment is not initialized yet.